### PR TITLE
Enable translations in store emulation

### PR DESCRIPTION
**Pull request by avstudnitz**
- Solves issue: If you use the Magento translation function `__` in an exported field, translation didn't work if you started the export from the command line.
- Tested with Magento editions/versions: 2.3.3 Commerce
- Tested with PHP versions: 7.1.26

### CODE REVIEW

File: src/Model/StoreEmulation.php [hunk 1]
```diff
@@ -5,22 +5,32 @@
...
         $this->emulation = $emulation;
+        $this->areaList = $areaList;
     }
 
     public function runInStore(int $storeId, callable $proceed)
     {
         try {
             $this->emulation->startEnvironmentEmulation($storeId, Area::AREA_FRONTEND, true);
+
+            $area = $this->areaList->getArea(\Magento\Framework\App\Area::AREA_FRONTEND);
```
Comment (fmarangi) [inline src/Model/StoreEmulation.php]: Could you please use fully-qualified class names here?

Comment (avstudnitz) [reply src/Model/StoreEmulation.php]: Sure. Fixed.

**Comment by fmarangi** [comment]
Hi @avstudnitz thanks for your contribution! After a quick search I have a feeling that it could actually be a M2 bug

https://github.com/magento/magento2/blob/36ea828171ac3930af4890322c7a5b338d1a96ef/app/code/Magento/Store/Model/App/Emulation.php#L157

Here they forgot to pass the $force value. I haven't tested it though, so I can't be 100% sure. In case it's fixed in the core we would have a duplicate load. Shall we introduce this in our module or is it better to have a rewrite on the single project level? What do you think?

**Comment by avstudnitz** [comment]
It sure is a Magento bug. I'd fix it in the module because:
- The call is not expensive and is only executed once per store
- The fix doesn't break anything if the issue is fixed in the core
- I wouldn't rely on a pull request (which doesn't exist yet AFAIK) being merged soon
- It would also support "older" Magento versions.

**Pull request merged**